app/views/articles/comment.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= form_for(@comment) do |f| %>
+  <% if @comment.errors.any? %>
+    <h2>Errors:</h2>
+    <ul>
+      <% @comment.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+    </ul>
+  <% end %>
+
+  <%= f.label(:name, "Name:") %>
+  <%= f.text_field(:name) %>
+  <br/>
+    <%= f.label(:text, "Comment:") %>
+  <%= f.text_field(:text) %>
+  <br/>
+  <%= f.submit("Save") %>
+<% end %>
```

Comment:
@silvadaniele I'm not sure to see where this file is used.

This is a `view` named `comment.html.erb` in the directory `app/views/articles`.
But I don't see it rendered anywhere in the code.

If your app seems to work just fine anyway, in means that you can juste delete this file